runtime/src/iree/modules/hal/utils/BUILD

```diff
@@ -22,5 +22,6 @@ iree_runtime_cc_library(
         "//runtime/src/iree/hal",
         "//runtime/src/iree/modules/hal:types",
         "//runtime/src/iree/vm",
+        "//runtime/src/iree/vm:impl",
```

Comment:
impl shouldn't be used outside of vm/... - what header is getting pulled in here?

Reply:
ah - you want to drop the `#include "iree/vm/shims.h"` from buffer_diagnostics.c - iree/vm/api.h includes that

Reply:
Yeah that one looked weird. 

iree/runtime/src/iree/vm/shims.h is flagged as being in impl

Reply:
(sorry github didn't show me your message before I responded)